autogen/json2class.cpp

```diff
@@ -1640,7 +1691,7 @@ void file_python_class(const NameDeps &obj, const std::string &filePythonCPP)
    cpp << "         python::init<\n";
    count = 0;
    for (auto &m : minfo) {
-      cpp << "            const " << m.fullVarType;
+      cpp << "            const " << (m.isDefaulted ? m.varType : m.fullVarType);
```

Comment:
Is this because Python doesn't know about the C++ class Defaulted?

Reply:
Yes. It's one of the reasons I had you add those constructors not using Defaulted. The init here uses those.